specs/state_machine.js

```diff
@@ -17,6 +17,17 @@ createMachine(
         },
       },
       AUTHORIZATION_COMPLETED: {
+        entry: assign({ auth_outcome: "OK" }),
```

Comment:
given that the sequence diagram changes between v1 and v2, maybe it's better to split the two diagrams?

Reply:
Good idea 👍🏻

Reply:
Fixed with c6e8c869fa14c39ce89afcad9a06027241ca4e60 🚀